test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+	@Test
+	public void testIsAnyNull() {
```

Comment:
You can omit the `test` prefix in the name because we already have a `@Test` annotation.